test/ectest.c

```diff
@@ -1441,75 +1432,6 @@ static int nistp_single_test(int idx)
     BN_CTX_free(ctx);
     return r;
 }
-
-/*
- * Tests a point known to cause an incorrect underflow in an old version of
- * ecp_nist521.c
- */
-static int underflow_test(void)
```

Comment:
TIL -- thanks for the tip :) I this case, I'd prefer it internal because (as I'm sure you know!) it's pretty common for developers to look at the tests to understand what they're supposed to do. And I wouldn't want to mislead anyone ;)